### PR TITLE
i155: Clean out old navigation stuff

### LINKED ISSUE #155: Clean out old navigation stuff
- [ ] also: rename AuthOrMain to InitialPage

### PR DESCRIPTION
**Pull request by nickmeinhold**
EDIT: I forgot - this branch already has #198 merged into it, so it'll be easier to review #198 and merge it into dev then update this branch before the review 

fixes #155 

Just a massive deletion session really... super satisfying.  If anything needs some explanation please let me know but hopefully it'll be a simple one (even though there's a lot of files)

### CODE REVIEW

**Review by tamari-gray — APPROVED** [review]
nice bug fix! and hello navigator 2!!

**Comment by tamari-gray** [comment]
good job!

File: lib/extensions/models_extensions.dart [hunk 1]
```diff
@@ -20,8 +20,8 @@ import 'package:flutter/material.dart';
 /// gone with a big map for now and may come back to optimize in future.
 extension NavigatorEntriesExt on BuiltList<PageData> {
   static final Map<PageData, MaterialPage> _pagesMap = {
-    InitialPageData(): MaterialPage<AuthOrMain>(
-        key: ValueKey(AuthOrMain), child: AuthOrMain()),
+    InitialPageData(): MaterialPage<InitialPage>(
+        key: ValueKey(InitialPage), child: InitialPage()),
```
[the remark this answers is not in the document]
Comment (nickmeinhold) [reply lib/extensions/models_extensions.dart:24]: totally agree, thanks!

File: lib/utils/redux/services_bundle.dart [hunk 2]
```diff
@@ -36,34 +34,26 @@ class ServicesBundle {
     _firestoreSettings = firestoreSettings;
   }
 
-  /// Navigation Key
-  final GlobalKey<NavigatorState> _navKey;
-
   /// Services
   final AuthService _authService;
-  final NavigationService _navigationService;
```
[the remark this answers is not in the document]
Comment (nickmeinhold) [reply lib/utils/redux/services_bundle.dart:44]: Haha and goood riddance!

**Pull request merged**